# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini
# Commit: b8a405d2e140
# Full Hash: b8a405d2e140de87b7a8b06e95a04f984a8ddf0a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-30 21:19:54
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - Implement text-wrap: balance for nsBlockFrame reflow. r=emilio
#   
#   A simple form of balance for short blocks, implemented by incrementally reducing
#   the effective inline-size used during line-breaking, up to the point where an
#   extra line would be created.
# ==============================================================================

diff -r 2a7646e1a4a5 -r b8a405d2e140 testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini
--- a/testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini	Sat Sep 30 15:53:11 2023 +0000
+++ b/testing/web-platform/meta/css/css-text/parsing/text-wrap-valid.html.ini	Sat Sep 30 15:53:12 2023 +0000
@@ -1,39 +1,13 @@
 [text-wrap-valid.html]
-  expected:
-    if (os == "android") and fission: [OK, TIMEOUT]
   [e.style['text-wrap'\] = "wrap" should set the property value]
     expected: FAIL
 
   [e.style['text-wrap'\] = "nowrap" should set the property value]
     expected: FAIL
 
-  [e.style['text-wrap'\] = "balance" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "stable" should set the property value]
-    expected: FAIL
-
   [e.style['text-wrap'\] = "pretty" should set the property value]
     expected: FAIL
 
-  [e.style['text-wrap'\] = "initial" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "inherit" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "unset" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "revert" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "revert-layer" should set the property value]
-    expected: FAIL
-
-  [e.style['text-wrap'\] = "auto" should set the property value]
-    expected: FAIL
-
   [e.style['text-wrap'\] = "wrap auto" should set the property value]
     expected: FAIL
 